java/org/contikios/cooja/Simulation.java

```diff
@@ -927,13 +972,8 @@ public void run() {
         Simulation.this.notifyObservers(this);
       }
     };
-    if (!isRunning()) {
-    	/* Simulation is stopped, change speed immediately */
-    	r.run();
-    } else {
-    	/* Change speed from simulation thread */
-    	invokeSimulationThread(r);
-    }
+    /* Change speed from simulation thread */
```

Comment:
I think this comment can be removed.

Reply:
It is an old comment but I can remove it.